teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name, parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here, Kindly add `age`, `name`, and `parent_permission` variables to initialize method parameters beside specialization because this is the right way to inherit these variables.

[Hint]: the initialize method will be something like that:
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
Well resolved 👍🏽